src/bin/statinamic-setup.js

```diff
@@ -1,38 +1,71 @@
-import debug from "debug"
-import npmInstallPackage from "npm-install-package"
+import "babel-polyfill"
 
+import program from "commander"
+import color from "chalk"
+import { join } from "path"
+import fs from "fs-extra"
+
+import { prompt } from  "./utils/inquirer"
+import questions, { defaultAnswers } from "./data/questions"
+import template from "./data/template"
 import {
+  version as statinamicVersion,
   peerDependencies as peerDeps,
   optionalPeerDependencies as opPeerDeps,
 } from "../../package.json"
 
-const tobeInstalledDeps = {
-  ...peerDeps,
-  ...opPeerDeps,
-}
-
-const log = debug("statinamic:bin:setup")
-
-log("Installing required dependencies...")
-
-const depsMap = Object.keys(tobeInstalledDeps)
-  .reduce(
-    (deps, dep) => {
-      deps.push(`${ dep }@${ tobeInstalledDeps[dep] }`)
-      return deps
-    },
-    []
-  )
-npmInstallPackage(
-  depsMap,
-  {
-    saveDev: true,
-  },
-  (err) => {
-    if (err) {
-      throw err
-    }
+(function() {
```

Comment:
Ideally yes. But babel will transform async funtion and but it in the top,  before `babel-polyfill' import statement. This cause some reference error.  Maybe add a comment to clarify that?